styles/AdvancedEditor.css

```diff
@@ -172,6 +172,13 @@
   padding-bottom: 6px;
   text-align: left;
 }
+.ProseMirror .box-element .block-section_title h1 {
+  font-style: italic !important;
+  font-weight: 400 !important;
+  line-height: 24px; /* 114.286% */
+  letter-spacing: -0.369px;
+  font-size: 21px !important;
+}
```

Comment:
font-size should be decreased to nested sections

Reply:
I'm using the font size from design

Reply:
done